extensions/wikia/InsightsV2/models/InsightsQueryPageModel.php

```diff
@@ -102,7 +102,7 @@ private function createTitles( $res ) {
 
 		while ( $row = $dbr->fetchObject( $res ) ) {
 			if ( $row->title ) {
-				$title = Title::newFromText( $row->title, $row->namespace );
+				$title = Title::makeTitle( $row->namespace, $row->title );
```

Comment:
@lkucharczyk, there's no `makeTitle` method for `Title` class - https://www.mediawiki.org/wiki/Manual:Title.php

Reply:
What is https://github.com/Wikia/app/blob/e0c46aa7394318f1c88132aaf8f2e5f6521e573c/includes/Title.php#L306-L329 though?